src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js

```diff
@@ -40,15 +40,30 @@ define(['sulumedia/model/media'], function(Media) {
                     selectable: false
                 }
             }
-        };
+        },
+
+        MEDIA_LANGUAGE = 'mediaLanguage';
```

Comment:
Same as above, would place that in a constants variable, otherwise it could be confused with an event.